pkgs/development/libraries/libpulsar/default.nix

```
@@ -92,11 +76,11 @@ stdenv.mkDerivation rec {
   '';
 
   meta = with lib; {
-    homepage = "https://pulsar.apache.org/docs/en/client-libraries-cpp";
+    homepage = "https://pulsar.apache.org/docs/3.0.x/client-libraries-cpp/";
```

```suggestion
    homepage = "https://pulsar.apache.org/docs/3.1.x/client-libraries-cpp/";
```
why do the version numbers not match up? strange